portfolio/src/main/webapp/education.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Education & Activities</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+  </head>
+
+  <body bgcolor="AliceBlue"> 
+    <h3 align="center">Education & Activities</h3>
+
+    <ul class="info" align="left">
+      <li><a href="https://github.com/kofimeighan">GitHub</a></li>
+      <li><a href="https://www.linkedin.com/in/kofimeighan/">LinkedIn</a></li>
+      <li>(561) 613-1936</li>
+      <li>kofimeighan@google.com</li>
+    </ul>
+
+    <ul class="nav">
+      <li><a href="index.html">Home</a></li>
+      <li><a href="education.html">Education & Activities</a></li>
+      <li><a href="pictures.html">Film Pictures</a></li>
+      <li><a href="mixes.html">Music Mixes</a></li>
+    </ul>
+
+
+    <h2><u>Education</u></h2>
+      <span><img src="images/pclogo.png" 
+                 width=150px height=175px alt="Pine Crest Logo"></span>
+        <p><i><b>Pine Crest School</b></i> - Pine Crest School is a
+          private prepatory school in Fort Lauderdale, Florida.
+           I attended from August 2015 to May 2019.</p>
+
+      <span><img src="images/Columbia_Engineering_logo.png" 
+                 width=150px height=175px alt="Columbia University Crown">
+        <p><i><b>Columbia University</b></i> - I currently attend Columbia 
+          University in the City of New York with an anticipated major in 
+          Computer Science.My expected graduation date is May 2023.</p>
+
+    <h2><u>Activities</u></h2>
+      <h5>At Columbia I partake in a few clubs and organizations,
+          I have listed most of them below:</h5>
+        <span><img src="images/Bacch B.png" 
+                   width=150px height=175px alt="Bacchanal Logo"></span>
+          <p><i><b>Bacchanal</b></i> - I am a member of Columbia University's
```

Comment:
There are three line wrapping methods in the style guide. (https://google.github.io/styleguide/htmlcssguide.html#HTML_Line-Wrapping).

Try not to mix them. Please fix all "<img" line wrapping in this file.